test/api/v3/integration/groups/POST-groups_groupId_leave.js

```diff
@@ -0,0 +1,86 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /groups/:groupId/leave', () => {
+  let user;
+  let userToInvite;
+  let group;
```

Comment:
Why are these 3 `group` variables in this scope when they are used only in the guilds context? Shouldn't `group` be called `guild` as in the `party` tests. And do we need `groupName` and `groupType` to be variables when we are never asserting on them(the way the party name and type are just hardcoded in the `beforeEach`)?